gn3/computations/rust_correlation.py

```diff
@@ -0,0 +1,91 @@
+"""module contains code integration  correlation implemented in rust here
+
+https://github.com/Alexanderlacuna/correlation_rust
+
+"""
+
+import subprocess
+import json
+import os
+
+
+from gn3.computations.qtlreaper import create_output_directory
+from gn3.random import random_string
+from gn3.settings import CORRELATION_COMMAND
+from gn3.settings import TMPDIR
+
+
+def generate_input_files(dataset: list[str], output_dir: str = TMPDIR):
+    """function generates outputfiles and inputfiles"""
+
+    tmp_dir = f"{output_dir}/correlation"
+
+    create_output_directory(tmp_dir)
+
+    tmp_file = os.path.join(tmp_dir, f"{random_string(10)}.txt")
+
+    with open(tmp_file, "w", encoding="utf-8") as file_writer:
+
+        file_writer.write("\n".join(dataset))
+    return (tmp_dir, tmp_file)
+
+
+def generate_json_file(**kwargs):
```

Comment:
**issue:** preferably make the args explicit. Do you see them varying?  Something like:
```suggestion
def generate_json_file(dir: str, file_name:str, method: str, delimiter: str) -> str:
```

Reply:
they are constants